packages/api/tests/lambdas/test-cleanExecutions.js

```diff
@@ -1,58 +1,470 @@
+      t.false(pgPayloadsEmpty(massagedExecution));
+    }
+  });
+  let massagedEsExecutions = await t.context.searchClient.query({
+    index: t.context.esIndex,
+    type: 'execution',
+    body: {},
+    size: 30,
+  });
+  massagedEsExecutions.results.forEach((massagedExecution) => {
+    if (massagedExecution.updatedAt < nonCompleteExpiration.getTime() && massagedExecution.status !== 'completed') {
+      t.true(esPayloadsEmpty(massagedExecution));
+    } else if (massagedExecution.updatedAt < completeExpiration.getTime() && massagedExecution.status === 'completed') {
+      t.true(esPayloadsEmpty(massagedExecution));
+    } else {
+      t.false(esPayloadsEmpty(massagedExecution));
+    }
+  });
+  completeExpirationDays = 8;
+  nonCompleteExpirationDays = 7;
+
+  nonCompleteExpiration = getExpirationDates(
+    completeExpirationDays,
+    nonCompleteExpirationDays,
+    true,
+    true
+  ).nonCompleteExpiration;
+  completeExpiration = getExpirationDates(
+    completeExpirationDays,
+    nonCompleteExpirationDays,
+    true,
+    true
+  ).completeExpiration;
+  process.env.completeExecutionPayloadTimeout = completeExpirationDays;
+  process.env.nonCompleteExecutionPayloadTimeout = nonCompleteExpirationDays;
+
+  await handler();
+  await sleep(5000);
+  massagedPgExecutions = await Promise.all(
+    t.context.execution_cumulus_ids.map(
+      async (cumulusId) => await model.get(t.context.knex, { cumulus_id: cumulusId })
+    )
+  );
+
+  massagedPgExecutions.forEach((massagedExecution) => {
+    if (massagedExecution.updated_at < nonCompleteExpiration && massagedExecution.status !== 'completed') {
+      t.true(pgPayloadsEmpty(massagedExecution));
+    } else if (massagedExecution.updated_at < completeExpiration && massagedExecution.status === 'completed') {
+      t.true(pgPayloadsEmpty(massagedExecution));
+    } else {
+      t.false(pgPayloadsEmpty(massagedExecution));
+    }
+  });
+  massagedEsExecutions = await t.context.searchClient.query({
+    index: t.context.esIndex,
+    type: 'execution',
+    body: {},
+    size: 30,
+  });
+  massagedEsExecutions.results.forEach((massagedExecution) => {
+    if (massagedExecution.updatedAt < nonCompleteExpiration.getTime() && massagedExecution.status !== 'completed') {
+      t.true(esPayloadsEmpty(massagedExecution));
+    } else if (massagedExecution.updatedAt < completeExpiration.getTime() && massagedExecution.status === 'completed') {
+      t.true(esPayloadsEmpty(massagedExecution));
+    } else {
+      t.false(esPayloadsEmpty(massagedExecution));
+    }
+  });
+  process.env = env;
 });
 
-test.serial('Function throws error if passed invalid complete timeout', async (t) => {
-  process.env.nonCompleteExecutionPayloadTimeout = 100;
-  process.env.completeExecutionPayloadTimeout = 'notaninteger';
-  let actual;
-  await cleanExecutionPayloads(executionModel).catch((error) => {
-    actual = error.message;
+test.serial('handler() throws errors when misconfigured', async (t) => {
```

Comment:
Nit: my preference would be to assert each known possible configuration issue separately. That's preference though so dealer's choice 😄